File: server/bcephal-dashboard/src/main/java/com/moriset/bcephal/dashboard/domain/DashboardReportFieldGroup.java
/**
 * 
 */
package com.moriset.bcephal.dashboard.domain;

/**
 * @author Joseph Wambo
 *
 */
public enum DashboardReportFieldGroup {

	ROW(1), COLUMN(2), COMMON(3), FILTER(4), GROUP(5);

	public int position;

	private DashboardReportFieldGroup(int position) {
		this.position = position;
	}

}

File: client/bcephal-client-model/Dashboards/DashboardReportFieldGroup.cs
﻿using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Dashboards
{
	[JsonConverter(typeof(StringEnumConverter))]
	public  enum DashboardReportFieldGroup
    {
		ROW,
		COLUMN,
		COMMON,
		FILTER,
		GROUP,
	}
}
